src/StackManager.js

```diff
@@ -182,16 +205,9 @@ export default class StackManager {
     const projectPath = this.getProjectPath();
     const localPath = ejectFilePath(projectPath, localFile);
 
-    return this.execDocker(`cp --follow-link ${container}:${remotePath} ${localPath}`);
-  }
-
-  // shell
-
-  shell(serviceName) {
-    const shell = this.stack.services.get(serviceName).shell;
-
-    if (!shell) throw new Error('This service does not have any shell.');
-
-    return this.spawnDockerCompose('exec', serviceName, shell);
+    return this.execDocker('cp', clean([
+      '--follow-link',
+      `${container}:${remotePath} ${localPath}`,
+    ]));
```

Comment:
- same here, we should use `clean` only when we have some optional arguments
- also, we can split the 2nd item and have `localPath` as the 3rd param